models/dodo/dodo_trades.sql

```diff
@@ -8,11 +9,11 @@
 }}
 
 {% set dodo_models = [
-ref('dodo_ethereum_trades')
-, ref('dodo_bnb_trades')
-, ref('dodo_polygon_trades')
-, ref('dodo_arbitrum_trades')
-, ref('dodo_optimism_trades')
+ref('dodo_pools_ethereum_trades')
```

Comment:
if you read directly from `dodo_pools_<blockchain>_trades` spells, then do we need the `dodo_<blockchain>_trades` spells at all? they seem to read from pools too